Makefile

```
@@ -1,4 +1,4 @@
-CARGO_BUILD_FLAGS = -Z build-std=core,alloc --target 3ds.json
+CARGO_BUILD_FLAGS = -Z build-std=core,alloc --target armv6k-nintendo-3ds
```

The makefile doesn't work if it builds core, but the files use std:
```suggestion
CARGO_BUILD_FLAGS = -Z build-std --target armv6k-nintendo-3ds
```

---

Thought I removed this, good catch!